generated/nidmm/nidmm.proto

```diff
@@ -53,7 +53,6 @@ service NiDmm {
   rpc ConfigureWaveformAcquisition(ConfigureWaveformAcquisitionRequest) returns (ConfigureWaveformAcquisitionResponse);
   rpc ConfigureWaveformCoupling(ConfigureWaveformCouplingRequest) returns (ConfigureWaveformCouplingResponse);
   rpc Control(ControlRequest) returns (ControlResponse);
-  rpc Control4022(Control4022Request) returns (Control4022Response);
```

Comment:
The PR description should definitely call out which obsolete deletions are intentional.

Reply:
Added a part at the top of the PR description for "intentional deletions" to list all of these out and the rationale for deleting them.